DQM/SiStripMonitorClient/scripts/LegendToQT.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python
+
+import Image, sys
+
+background = Image.open(sys.argv[1])
+foreground = Image.open(sys.argv[2])
+
+background.paste(foreground, (3000, 40), foreground)
```

Comment:
@davidlange6 the problem here is subtle, the complete Tracker Maps are never ROOT objects, they are build from root files of single layers then combined in a SVG image and then dumped as PNG, so there is no real easy way to add a root style legend there. The most straightforward way to do it is adding the png image in this way, and this require the image package to be available in python (the CMSSW python has not this package). However this part of the code is commented in the main script, will be uncommented by the shifters once a new release will be needed. 
In the case I am misunderstanding and you already know where a legend can be introduced using ROOT, please give us more indications and we will implement your way.